docs/cloudfoundry/cf-per-route-virtuals.rst

```diff
@@ -55,6 +55,14 @@ You can use as many Service Plans as you need to define the BIG-IP services your
 
    \
 
+   .. warning::
+
+      It's important to use the proper `CIDR format`_ when allocating IP addresses or networks for the :code:`tier2_ip_range` config parameter (for example, :code:`172.0.0.0/24` for IPv4). Failure to use the proper format may result in errors when the |cfctlr| configures BIG-IP objects.
+
+      See `Overview of the Standard Virtual Server`_ on AskF5 for more information. When allocating networks make sure that the network prefixes align with CIDR block boundaries. For example, :code:`10.105.175.245/30` will fail, since it is not referencing an address at the boundary. However, :code:`10.105.175.244/30` will succeed due to proper boundary alignment.
```

Comment:
```
When allocating networks, the network prefixes must align with CIDR block boundaries. For example:

- :code:`10.105.175.245/30` causes failures because it doesn't reference an address at the CIDR boundary. 
- :code:`10.105.175.244/30`, which correctly references an address at the CIDR boundary, succeeds.
  
See `Overview of the Standard Virtual Server`_ on AskF5 for more information.
```